optuna/storages/_rdb/storage.py

```diff
@@ -49,6 +50,17 @@
 _logger = optuna.logging.get_logger(__name__)
 
 
+def _ensure_not_nan(value):
+    # Ensure the value is not Nan, which is not supported by MySQL
+    # if Nan, change it the None
+    if isinstance(value, tuple) or isinstance(value, list):
```

Comment:
This can be simplified:
```suggestion
    if isinstance(value, (tuple, list)):
```